src/Angor/Client/Pages/Signatures.razor

```diff
@@ -560,12 +557,10 @@
         if (!InvestorTransactionActions.CheckInvestorRecoverySignatures(info, investorTrx, sig))
             throw new InvalidOperationException();
 
-        sig.SignatureType = SignatureInfoType.Recovery;
-
         return sig;
     }
-  
-    public class SignatureItem
+
+    public class SignatureRequest
```

Comment:
Why this rename? it was change to `SignatureItem`

Reply:
Fixed